src/main/webapp/app/shared/organization-selector/organization-selector.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import { OrganizationManagementService } from 'app/admin/organization-management/organization-management.service';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { TranslateService } from '@ngx-translate/core';
+import { JhiAlertService } from 'ng-jhipster';
+import { Organization } from 'app/entities/organization.model';
+import { Course } from 'app/entities/course.model';
+
+@Component({
+    selector: 'jhi-organization-selector',
+    templateUrl: './organization-selector.component.html',
+})
+export class OrganizationSelectorComponent implements OnInit {
+    organizations: Organization[];
+    availableOrganizations: Organization[];
+    course: Course;
```

Comment:
Maybe I missed it but do you use this or set it anywhere?

Reply:
The component is used in `course-update.component.ts` and `user-management-update.component.ts` as modal and currently assigned organizations are passed using `modalRef.componentInstance.organizations = this.user.organizations;` to the `organizations: Organization[]` property.

Reply:
No, I meant the course variable.

Reply:
My bad.. You are right (unused) :sweat_smile:

Reply:
Removed with c5025f2
Thanks :+1: